instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py

```diff
@@ -127,6 +133,7 @@ def wrap_connect_(
             connection_attributes=connection_attributes,
             version=version,
             tracer_provider=tracer_provider,
+            capture_parameters=capture_parameters,
         )
         return db_integration.wrapped_connection(wrapped, args, kwargs)
 
```

Comment:
I believe `instrument_connection` needs to have this parameter as well.

Reply:
makes sense.